supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -576,9 +576,16 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     }
   }
 
-  private def handleSuccess(outputMappings: Try[JobOutputs], returnCode: Int, executionHandle: ExecutionHandle): ExecutionHandle = {
+  private def gatherJobOutputFiles: Map[String,String] = {
+      val callOutputFiles = List(jesCallPaths.stderrPath, jesCallPaths.returnCodePath, jesCallPaths.stdoutPath, gcsExecPath)
+      val mapOfFiles = callOutputFiles map { case(path) => path.getFileName.toString -> path.toString } toMap
+      val result = mapOfFiles ++ Map("callRootPath" -> callRootPath.toString)
```

Comment:
sending it along since I use it to resolve paths when JesCacheHitCopyingActor is doing its job. I can do something less risky if requested....